core/arch/arm/kernel/kern.ld.S

```diff
@@ -250,6 +268,10 @@ SECTIONS
 		__text_init_end = .;
 	}
 
+	. = ALIGN(SMALL_PAGE_SIZE);
```

Comment:
True. An alternate implementation of this P-R simply splits read-only and red-write memory. It can be easily done and could save some kbytes (dear pager!). But when pager is disabled, we don't expect to struggle for few kBytes.

I would not like a new configuration switch as CFG_CORE_MAPS_READONLY_EXECUTABLE.
It's about reducing attack surface. Should we cowardly restrict this, or nicely waste memory few maybe-useless extra hardening?